resources/extendedcrafting/lang/en_us.lang

```diff
@@ -0,0 +1,3 @@
+tile.ec.table_basic.name=Engineering Table
+container.ec.table_basic=Engineering
+jei.ec.table_crafting_3x3=Engineering
```

Comment:
Other lang files have new line separator here, but this one doesnt. Please keep consintency of lang files for easier syncing translations in future.